recipe/build.sh

```diff
@@ -11,19 +11,27 @@ for i in `ls`; do
     [[ $i == "build_env_setup.sh" ]] && continue
     [[ $i == "conda_build.sh" ]] && continue
     [[ $i == "metadata_conda_debug.yaml" ]] && continue
-    if [[ $i == "lib" ]] || [[ $i == "include" ]]; then
+    if [[ $i == "bin" ]]; then
```

Comment:
I would make the below `elif` clause instead just an `if` that also includes `bin`, then put the `bin`-specific logic into a nested `elif` like the `$i == "lib"` block. That maintains the existing structure and reduces duplication (the below three lines become superfluous).

Reply:
Removed the `if [[ $i == "bin" ]]` and rolled it into the blocker underneath.